<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QUaUserWidgetEdit</class>
 <widget class="QWidget" name="QUaUserWidgetEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelUserName">
     <property name="text">
      <string>User Name : </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditUserName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>Password : </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditPass">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelHash">
     <property name="text">
      <string>Hash : </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPlainTextEdit" name="plainTextEditHash"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelRole">
     <property name="text">
      <string>Role : </string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxRole"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
